docs/azure/deployment.md

```diff
@@ -17,12 +17,12 @@ Whether your workflow is through the [Azure CLI](https://learn.microsoft.com/cli
 
 These tutorials from Microsoft Learn describe different ways of creating and deploying apps to Azure via Visual Studio Code:
 
-Tutorial | Description
---- | ---
-[Deploy Azure Functions](https://learn.microsoft.com/azure/developer/javascript/tutorial/azure-function-cosmos-db-mongo-api) | Build and manage Azure Functions serverless apps directly in VS Code with the Azure Functions extension.
-[Deploy using the App Service extension](https://learn.microsoft.com/azure/app-service/tutorial-nodejs-mongodb-app) | Manage Azure resources directly in VS Code with the Azure App Service extension.
-[Deploy using Docker](https://learn.microsoft.com/azure/developer/javascript/tutorial/tutorial-vscode-docker-node/tutorial-vscode-docker-node-01) | Deploy your website using a Docker container.
-[Deploy a static website](https://learn.microsoft.com/azure/static-web-apps/getting-started) | Create, deploy, and update a static website.
+Tutorial(s) | Description | Framework / Language
+--- | --- | ---
+[Deploy a static website](https://learn.microsoft.com/azure/static-web-apps/getting-started) | Create, deploy, and update a static website | Angular, Blazor, React, or Vue
```

Comment:
Should this be Vue.js?

Reply:
maybe! The reason I didn't say Vue.js is because the tab for it in the tutorial (https://learn.microsoft.com/en-us/azure/static-web-apps/getting-started?tabs=vue) just lists it as "Vue"

Reply:
I usually check the official site to for technology naming. https://vuejs.org/

Reply:
Makes sense, it looks like they refer to it as Vue in their docs